web_monitoring/db.py

```diff
@@ -163,6 +168,22 @@ def from_env(cls):
 Alternatively, you can instaniate Client(user, password) directly.""")
         return cls(email=email, password=password, url=url)
 
+    def request(self, method, url, data=None, raw_response=False, **kwargs):
+        if data is not None:
+            headers = kwargs.setdefault('headers', {})
+            if isinstance(data, collections.Sequence):
```

Comment:
Can you please use `collections.abc.Sequence` instead? `collections.Sequence` is deprecated and fires off warnings in tests. It’s going to be removed in Python 3.9.

Reply:
Done